praw/models/reddit/subreddit.py

```diff
@@ -45,22 +45,22 @@ class Subreddit(
 
     .. code-block:: python
 
-       subreddit = reddit.subreddit('redditdev')
+       subreddit = reddit.subreddit("redditdev")
 
     While ``r/all`` is not a real subreddit, it can still be treated like
     one. The following outputs the titles of the 25 hottest submissions in
     ``r/all``:
 
     .. code-block:: python
 
-       for submission in reddit.subreddit('all').hot(limit=25):
+       for submission in reddit.subreddit("all").hot(limit=25):
            print(submission.title)
 
     Multiple subreddits can be combined with a ``+`` like so:
 
     .. code-block:: python
 
-       for submission in reddit.subreddit('redditdev+learnpython').top('all'):
+       for submission in reddit.subreddit('redditdev+learnpython').top("all"):
```

Comment:
Some of the stuff like this with inconsistent quoting on the same line would also be nice to fix.

Reply:
I'm pushing another commit for this